### DISCUSSION TITLE
Can `age` be built with a newer Go version?

### DISCUSSION THREAD

**Discussion by Tang8330** [FiloSottile/age#549] [General]
`age` is built with `go 1.19` which has quite a bit of vulns.

Picked up by Docker Scout:

![image](https://github.com/FiloSottile/age/assets/4412200/bd979fc8-41fd-4c8a-9a75-1ba18ee8c774)

**Comment by cipriancraciun** [comment]
I've just tried to `go build ./cmd/age` on the latest `main` branch with Go `1.21.7` and it seems that the build process works without errors or warnings, and at lest the resulting binary is able to encrypt.

Did you encounter any issues when building with newer versions of Go?

**Comment by Tang8330** [comment]
@cipriancraciun I didn't encounter an error -- was mostly asking to see if we can get a new build added to the releases page: https://github.com/FiloSottile/age/releases
